Kino/MainScreen/View/FavouritesViewController.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  FavouritesViewController.swift
+//  Kino
+//
+//  Created by Viktoriya Polozova on 15/08/2024.
+//
+
+import UIKit
+
+class FavouritesViewController: UIViewController {
```

Comment:
favorites

Reply:
favourites in British / favorites in American